qa/t/lib/QA/Util.pm

```diff
@@ -452,11 +452,14 @@ sub check_page_load {
     }
   }
 
-  if ($expected_uri->query_param('id')) {
-    if ($expected_uri->query_param('id') eq '__BUG_ID__') {
-      $uri->query_param('id' => '__BUG_ID__');
-    }
+  if ($expected_uri->query_param('id') and $expected_uri->query_param('id') eq '__BUG_ID__') {
+    $uri->query_param('id' => '__BUG_ID__');
+  }
+
+  if ($expected_uri->query_param('list_id') and $expected_uri->query_param('list_id') eq '__LIST_ID__') {
```

Comment:
Tests were crashing because they expected a hard coded listid (which apparently is generated on every request and not tied to a specific/saved list). I believe it is appropriate to ignore the actual value of the list_id param, hence this function.